app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -45,35 +45,62 @@ class LuciBuildService {
     this.log = log;
   }
 
-  /// Returns a BuildBucket build for a given Github [slug], [commitSha] and
+  /// Returns an Iterable of try BuildBucket build for a given Github [slug], [commitSha],
   /// [builderName].
-  Future<Build> getTryBuild(
+  Future<Iterable<Build>> getTryBuilds(
     github.RepositorySlug slug,
     String commitSha,
     String builderName,
+  ) async {
+    final Map<String, List<String>> tags = <String, List<String>>{
+      'buildset': <String>['sha/git/$commitSha'],
```

Comment:
Should we explicitly add a tag for differentiate our try and prod builds?

Reply:
we usually use the bucket for that pupose.